components/Contributors.js

```diff
@@ -1,28 +1,33 @@
 export default function Contributors() {
-  /* 
-  
-  Add your name and Github here! 
-  
+  /*
+
+  Add your name and Github here!
+
   */
   const people = [
     { name: "Sam", githubLink: "https://github.com/SamSamskies" },
     { name: "Mark", githubLink: "https://github.com/MarkWillisford" },
     { name: "Levon", githubLink: "https://github.com/itfibonacci" },
     { name: "Karen", githubLink: "https://github.com/karleee" },
     { name: "James", githubLink: "https://github.com/ngjamesng" },
-    { name: "Megan", githubLink: "https://github.com/mcdevittbass"}
+    { name: "Megan", githubLink: "https://github.com/mcdevittbass" },
+    { name: "Emily", githubLink: "https://github.com/eaquin1" }
```

Comment:
I think you forgot to run Prettier on this change. Prettier adds trailing commas by default. You should be able to configure your editor to format on save.